tests/test_schema_validation.py

```diff
@@ -209,6 +209,19 @@ def test_invalid_survey_id_whitespace(self):
 
         self.assertEquals(errors.get('message'), "'lms ' does not match '^[0-9a-z]+$'")
 
+    def test_invalid_routing_when_answer_count(self):
+        """Asserts that invalid `when` routing_rules are caught for `answer_count`"""
+        file = 'schemas/test_invalid_routing_when_answer_count.json'
```

Comment:
Nitpick - this overrides the builtin `file` function.

Reply:
I just followed the convention in from existing tests - I can change them all in this PR, or do that in a separate refactor one?